app/models/notifications/mentioned.rb

```diff
@@ -13,19 +13,26 @@ def deleted_translation_key
     end
 
     def linked_object
-      target.post
+      target.mentions_container
     end
 
     def self.notify(mentionable, recipient_user_ids)
       actor = mentionable.author
+      relevant_mentions = filter_mentions(
+        mentionable.find_or_create_mentions.local.where.not(person: mentionable.author),
```

Comment:
why did you change this to `find_or_create_mentions`? The `mentions` should always be there at this point, if you create them in the `after_create`. The problem with `find_or_create_mentions` is, that this always parses the whole text again, if the post has NO mentions, because `mentions` would ALWAYS be empty for this posts, so it tries to create it. And `after_create` does this once. (same goes for comments without mentions)